.github/workflows/build.yml

```diff
@@ -64,21 +64,22 @@ jobs:
       if: ${{ runner.os == 'Linux' && matrix.node_version == 12 }}
       run: |
         npm run cov && npm run cov:report
-    - name: Coveralls
-      if: ${{ runner.os == 'Linux' && matrix.node_version == 12 }}
-      uses: coverallsapp/github-action@master
-      with:
-        github-token: ${{ secrets.GITHUB_TOKEN }}
+    # Coveralls is down for maintenance, disable it for now
```

Comment:
We shouldn't disable it at the script, we could just merge the PR.

Reply:
The ci will fail If enable it, and we can't run the other checkings.